demo/activitychart_basic_demo.html

```diff
@@ -13,6 +13,9 @@
     <link rel="import" href="../activitychart-basic.html">
 
     <style is="custom-style" include="demo-pages-shared-styles">
+      div.vertical-section-container {
+        max-width: 1500px;
```

Comment:
seems to not be working on Mac w/ Firefox though, couldnt find a reason why... see #25